packages/api/tests/endpoints/test-granules.js

```diff
@@ -737,15 +639,13 @@ test('DELETE returns 404 if granule does not exist', async (t) => {
   t.true(response.body.message.includes('No record found'));
 });
 
-test('DELETE deleting an existing granule that is published will fail and not delete records', async (t) => {
-  const { s3Buckets, newGranule } = await createGranuleAndFiles({
+test.serial('DELETE deleting an existing granule that is published will fail and not delete records', async (t) => {
```

Comment:
This is hopefully temporary. I'm seeing a failure in CI that I don't see locally in these tests. I'm trying to run these in serial to see if that's the issue. If it is, there might be something in the way I'm creating test granules that needs to change.

Reply:
This does seem to alleviate the test failure in CI (though they are still failing due to the test helper mentioned below). I'll see if I can get these running in parallel.

Reply:
I'm assuming this still needs to be addressed, will comment below

Reply:
Yeah, I've been poking at this a bit. I'm thinking these might legitimately need to be run in serial. There are 4 tests here for delete that each create Dynamo, Postgres, and S3 buckets and records and delete all of it. I'm wondering if the problem is that running these in parallel causes issues where one test expects the state of the tables one way but another test is inserting/deleting records. Particularly this test highlighted where delete fails...

I'm tempted to keep these in serial. Conceptually it sorta makes sense to me. What do you think?